app/apps/client/gameCenter/apiHandler.js

```diff
@@ -0,0 +1,52 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
I'd say you could move this file to `app/apps/client/extrernalComponent`; it seems more appropriate and easier to find when we have to change it later on